fuel-txpool/src/service.rs

```diff
@@ -10,9 +13,11 @@ pub struct ServiceBuilder {
     sender: txpool::Sender,
     receiver: mpsc::Receiver<TxPoolMpsc>,
     config: Config,
-    broadcast: broadcast::Sender<TxStatusBroadcast>,
+    consumer: broadcast::Sender<TxStatusBroadcast>,
```

Comment:
consumer could mean a lot of things... we could just call this `tx_status_updates`?

Reply:
I agree. I am aiming to disambiguate some of the naming: `sender`,  `receiver`, `consumer`, etc., are pretty vague and I find it can be confusing trying to understand which channels are used for what. I am taking a stab at that now.